.emacs.d/init.el

```diff
@@ -109,6 +116,10 @@ Despite this, your config appears to have loaded successfully.")
   (if loaddefs-buffer
       (kill-buffer loaddefs-buffer)))
 
+;; If in dev mode, load the test runner.
+(when kotct/dev-mode
+  (require 'dot-tests "~/.emacs.d/test/lisp/dot-tests"))
```

Comment:
This will be a good enough first step, I guess.  We can go ahead and merge, and if it causes problems we can rethink.